sigh/lib/sigh/resign.rb

```diff
@@ -21,6 +21,16 @@ def self.resign(ipa, signing_identity, provisioning_profiles, entitlements, vers
 
     def resign(ipa, signing_identity, provisioning_profiles, entitlements, version, display_name, short_version, bundle_version, new_bundle_id, use_app_entitlements, keychain_path)
       resign_path = find_resign_path
+
+      if keychain_path
+        keychain_path = File.expand_path(keychain_path)
```

Comment:
expanding the keychain path will have no negative influence on the already present uses of that variable further down? (e.g. line 42(old)/52(new))